web_responsive/views/web.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+
+    <template id="webclient_bootstrap"
+              inherit_id="web.webclient_bootstrap"
+              name="App Drawer - Web Client"
+              >
+
+        <xpath expr="//div[@class='o_sub_menu']" position="replace" />
```

Comment:
Looks like Odoo added some linting to the Qweb:

>2017-10-07 08:32:01,969 150 WARNING openerp_test odoo.addons.base.ir.ir_ui_view: Error-prone use of @class in view App Drawer - Web Client (): use the hasclass(*classes) function to filter elements by their classes

Simple change, because it seems they also added a `hasclass` shortcut:
```
//div[@hasclass('o_sub_menu')]
```

vs it used to be:

```
//div[@contains('class', 'o_sub_menu')]
```

Reply:
IMO we should port this change backwards too